static/js/redux/ui/top_bar_advising.jsx

```diff
@@ -78,6 +79,28 @@ class TopBarAdvising extends React.Component {
   }
 
   render() {
+    const ReturnToScheduleButton = (
+      <div className="cal-btn-wrapper">
+        <a href="/">
+          <button
+            className="import-data"
```

Comment:
you will want to change this CSS class to a different name now that it isn't the import data button anymore.

Reply:
It's really strange, but without naming it 'import-data', the button does not snap to the right side